projects/fourier/lib/fourier/utilities/swift_package_manager.rb

```diff
@@ -15,12 +16,23 @@ def self.build_product(product)
         )
       end
 
-      def self.build_fat_release_library(path:, product:, output_directory:, swift_build_directory:)
+      def self.build_fat_release_library(
+        path:,
+        product:,
+        output_directory:,
+        swift_build_directory:,
+        xcode_version:,
+        xcode_version_libraries:
+      )
+        xcode_path = path_to_xcode(xcode_version)
+        xcode_path_frameworks = path_to_xcode(xcode_version_libraries || xcode_version)
+
         Dir.chdir(path) do
-          # Libraries needs to be build with a different Xcode version, defined in .xcode-version-libraries
-          Utilities::System.system(
-            "sudo xcode-select -switch /Applications/Xcode_$(<.xcode-version-libraries).app"
-          )
+          if xcode_path_frameworks != path_to_xcode
```

Comment:
Is `path_to_xcode` invoking the method? Not a ruby expert, is it possible to do `path_to_xcode()` to make it clearer?

Reply:
I initially used parenthesis, but my linter told me that's not how you do it in Ruby apparently. I'm no ruby expert either, so I just listened to the linter. 😅